src/type-converter.js

```diff
@@ -12,6 +12,8 @@ const { TextDecoder, TextEncoder } = require("text-encoding");
 const typeConversions = {
   "&[u8]": {
     /**
+     * Copy &[u8] arguments into shared memory before passing to Rust
```

Comment:
Thanks for adding these docs! Can you put "&[u8]" in back ticks to highlight it as code?